### PR TITLE
[Windows] Update npm parcel package to v2

### PR DESCRIPTION
**Pull request by V-Zabayrachny**
# Description
This PR updates the npm parcel package to v2

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by miketimofeev — APPROVED** [review]

**Review by miketimofeev — CHANGES_REQUESTED** [review]
@V-Zabayrachny I couldn't find parcel in the readme file. I believe we have to add it to our software report.

**Review by GITHUBGUY12357999 — CHANGES_REQUESTED** [review]
requesting changes

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 1]
```diff
@@ -56,6 +56,7 @@ $packageManagementList = @(
     (Get-HelmVersion),
     (Get-NPMVersion),
     (Get-NugetVersion),
+    (Get-ParcelVersion),
```
Comment (miketimofeev) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:59]: - [Parcel is a zero configuration build tool for the web](https://github.com/parcel-bundler/parcel), not the package management utility
- We don't install parcel for windows-2022.

Comment (GITHUBGUY12357999) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:59]: ok

Comment (V-Zabayrachny) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Thanks. I fixed it.

**Review by al-cheb — DISMISSED** [review]

**Review by vsafonkin — DISMISSED** [review]

**Review by dmitry-shibanov — DISMISSED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 2]
```diff
@@ -174,6 +174,11 @@ function Get-YarnVersion {
     return "Yarn $yarnVersion"
 }
 
+function Get-ParcelVersion {
+    $parcelVersion = parcel --version
+    return "Parcel $parcelVersion"
+}
+
```
Comment (miketimofeev) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:181]: This is linux script, not the windows

Comment (V-Zabayrachny) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1]: I'm sorry. I fixed it.

**Review by miketimofeev — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**